dhall/src/Dhall/Import.hs

```diff
@@ -766,10 +767,25 @@ loadWith expr₀ = case expr₀ of
 
                     expr' <- loadDynamic `catches` [ Handler handler₀, Handler handler₁ ]
 
+                    -- Make current node the dot graph
+                    zoom dot (State.put $ importNode here)
+
                     zoom stack (State.put imports')
                     expr'' <- loadWith expr'
                     zoom stack (State.put imports)
 
+                    zoom dot (State.modify $ \getSubDot -> do
+                        parentNode <- _dot
+
+                        -- Get current node back from sub-graph
+                        hereNode <- getSubDot
+
+                        -- Add edge between parent and here
+                        () <- parentNode .->. hereNode
```

Comment:
Minor suggestion: I don't think the `() <-` is necessary

Reply:
Ah yeah of course :man_facepalming: